src/app.js

```diff
@@ -797,7 +796,27 @@ async function processOne(hook) {
 				await processHook(hook)
 				break
 			}
+			case 'max_battle': {
+				if (config.general.disableMaxBattle) {
+					fastify.controllerLog.debug(`${hook.message.id}: MaxBattle was received but set to be ignored in config`)
+
+					break
+				}
+				if (!hook.message.poracleTest) {
+					fastify.webhooks.info(`max_battle ${JSON.stringify(hook.message)}`)
+					const cacheKey = `${hook.message.id}${hook.message.battle_end}${hook.message.battle_pokemon_id}`
```

Comment:
The cache will expire well ahead of the max battle most probably, so this may not provide any value whatsoever.
What causes golbat to resend a station webhook and do we care that this will generate a new alert?

Reply:
hook.message.id -> Station ID
hook.message.battle_end -> is eqal to end_time in golbat DB
hook.message.battle_pokemon_id -> if this change we get a new alert on same station

Why the cache should expire in normal use ahead of the max battle end?
But on Maxmondays or Events, it should gen. a new alert.

Reply:
Yes, the cache lasts an hour (I think). And the max battle has a duration much longer than that, so updates within the period will not be eliminated by the cache

Reply:
> Rephrasing my question - are there any duplicate webhook messages that you are trying to avoid passing into the alerting engine? 
Maxbattles have rotating moves and there are stationed mons which change, so i wanted to be sure, not to gen unnecessary alterts. But maybe im wrong with my thought.

Reply:
Hmm... As the first webhook could be sent the day before, we might still send another hook at 7PM... If I think about it, that's not that bad I guess

Do you have some notification examples for max monday or e.g. this event with Toxtricity?! :)